app/src/main/res/values/styles.xml

```diff
@@ -11,6 +11,13 @@
     <item name="android:actionBarTabTextStyle">@color/white</item>
   </style>
 
+  <style name="SettingsTheme" parent="Theme.AppCompat.NoActionBar">
```

Comment:
use text color for secondary color and put these added lines in the last. Rest 👌